migrations/versions/0388_populate_letter_branding.py

```diff
@@ -7,8 +7,8 @@
 """
 from alembic import op
 
-revision = '0388_populate_letter_branding'
-down_revision = '0387_migrate_alt_text'
+revision = "0388_populate_letter_branding"
```

Comment:
Is something flagging this formatting as invalid? Everything passed for me when I added this file because we exclude the migrations directory from black [here](https://github.com/alphagov/notifications-api/blob/c917176a9b6a1401a8541b52d813272d65852159/Makefile#L72) 🤔

Reply:
Ah, yeah, this change is coming from `pre-commit`, which doesn't have the `extend-exclude`. Looking at the best way to deal with it...

Reply:
Although I think we actually just shouldn't have migrations excluded, it seems like everything is mostly formatted correctly there - except the latest migrations.

So just going to drop that exclusion...